### ISSUE TITLE
combining the branches master and configNumericInputs

### ISSUE THREAD

**Issue by 0ldM4j0r** [PoonLab/KaphiShiny#9]
Now that I can generate inputs dynamically in configNumericInputs. I should pass said inputs to the server to initialize a config, from there I should process the data as in the master branch.

**Comment by 0ldM4j0r** [comment]
I solved the second error by pressing on the `Proposals` tab before pressing the `Initialize Priors & Proposals` button, this results in the `Proposals` inputs being initialized and therefore the `get1index` error is gone, as it was thrown due to not being able to access the `Proposals` inputs due to them not existing.

**Comment by 0ldM4j0r** [comment]
I solved the first error b979471, but now an error is thrown when sampling priors:
``` R
Warning: Error in sample.priors: ERROR: argument must be an smc.config object
Stack trace (innermost first):
    70: sample.priors
    69: FUN [/home/fabusard/git/KaphiShiny/app.R#457]
    68: lapply
    67: sapply
    66: rbind
    65: observeEventHandler [/home/fabusard/git/KaphiShiny/app.R#457]
     1: runApp
```
when checking the config before setting the model it appears that the function `distribution.parameters` didn't work as expected:
```
$params
[1] "lambda"

$priors
$priors$lambda
[1] "rgamma(n=1,rate=,shape=)"


$prior.densities
$prior.densities$lambda
[1] "dgamma(arg.prior,rate=,shape=)"


$constraints
NULL

$proposals
$proposals$lambda
[1] "rnorm(n=1,mean=,sd=)"


$proposal.densities
$proposal.densities$lambda
[1] "dnorm(arg.delta,mean=,sd=)"


$model
[1] NA

$nparticle
[1] 1000

$nsample
[1] 5

$ess.tolerance
[1] 1.5

$final.epsilon
[1] 0.01

$final.accept.rate
[1] 0.015

$quality
[1] 0.95

$step.tolerance
[1] 1e-05

$dist
[1] "kernel.dist(x,y,decay.factor=0.2,rbf.variance=100.0,sst.control=1.0)"

$decay.factor
[1] 0.2

$rbf.variance
[1] 100

$sst.control
[1] 1

$norm.mode
[1] "NONE"
```

**Comment by 0ldM4j0r** [comment]
Solved the above error with config 9783a6c42d22f27cbc165665bae52a1049ca352e 231fb37da548a526249881a56250ee6d426603b8

**Comment by 0ldM4j0r** [comment]
Merge was successful bc3f51a2c94c9132257ed867e3d1016691f76f76
